index.html

```diff
@@ -254,10 +263,10 @@ <h3>2015</h3>
   <section class="contact-me">
     <div class="head-section-co">
       <div id="head-container-co">
-        <h1 class="head-co">
+        <h2 class="head-co">
           About, <br />
           Myself
-        </h1>
+        </h2>
```

Comment:
Great work @obibaadoma  👍, Kindly give this 👆 head tag the correct style.
**Your design**
![wrongabt](https://user-images.githubusercontent.com/73854999/199366308-baab4e03-3468-477e-b35f-7c2474e4f0e1.jpg)

**Figma design**
![abt](https://user-images.githubusercontent.com/73854999/199366224-25af5911-7c03-4f60-a81d-4dae19870cff.jpg)

Reply:
fixed the design